src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/GetAzureStorSimpleResource.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Management.Automation;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.WindowsAzure.Commands.StorSimple.Properties;
+
+namespace Microsoft.WindowsAzure.Commands.StorSimple.Cmdlets
+{
+    /// <summary>
+    /// this commandlet returns all resources available in your subscription
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "AzureStorSimpleResource"), OutputType(typeof(IEnumerable<ResourceCredentials>), typeof(ResourceCredentials))]
+    public class GetAzureStorSimpleResource : StorSimpleCmdletBase
+    {
+        [Alias("Name")]
```

Comment:
We are using the alias "Name" to denote the name of the entity acted upon by the cmdlet. For example, in case of resource related cmdlets it is the ResourceName, for device cmdlets it is DeviceName where for volume cmdlets it is VolumeName. Hope that makes sense.